coderkevin/fresh-data-wp-plugin-example
# Add wp-rest-api and tags list

This adds category and tag api endpoints, and a table of tags for
display.

To Test:
0. Clone `fresh-data` and do `npm install` and `npm link`.
1. Clone this repo under wp-content/plugins in a WP dev install.
2. `npm install`
3. `npm link fresh-data`
4. `npm start`
5. `Enable the plugin`
6. `Visit the "Fresh Data 🥕" admin menu item.`
7. Verify you see the "Tags" heading and the list of tags from you site appear underneath it within a couple of seconds.
